waterbox/opera/Makefile

```diff
@@ -0,0 +1,107 @@
+CCFLAGS := 	-I. \
+											 -Iopera-libretro \
+											 -Iopera-libretro/libopera \
+											 -Iopera-libretro/libretro-common/include \
+											 -Iopera-libretro/deps \
+											 -Iopera-libretro/deps/lzma-19.00/include \
+											 -Iopera-libretro/deps/libchdr/include \
+											 -Iopera-libretro/deps/zlib-1.2.11 \
+										  -DHAVE_CHD \
+												-D_7ZIP_ST \
+												-DDR_FLAC_NO_STDIO \
+												-D__LIBRETRO__ \
+												-DNDEBUG \
+												-DHAVE_CDROM \
+												-DHAVE_STDINT_H \
+												-DHAVE_STDLIB_H \
+												-DHAVE_SYS_PARAM_H \
+												-DINLINE=inline \
+												-DTHREADED_DSP
+
+CXXFLAGS := $(CCFLAGS) 	-std=c++20

```

Comment:
Is this just for the bizhawk.cpp? Might be easier to just make it a normal `.c` file and get rid of all the cpp handling.

Reply:
I do use std::string in bizhawk.cpp. It is much cleaner than C-style string manip. But I removed the c++20 and cleaned things up